.travis.yml

```diff
@@ -14,38 +12,50 @@ python:
   - "nightly"
 
 
+env:
+  global:
+    - ANYBLOK_DATABASE_NAME=travis_ci_test
+    - ANYBLOK_DATABASE_DRIVER=postgresql
+  matrix:
+    - ANYBLOK_DATABASE_USER=postgres
+
 matrix:
+  include:
+      - python: "pypy3.5"
+        env: ANYBLOK_DATABASE_DRIVER=postgresql+psycopg2cffi
+        addons:
+            postgresql: "9.6"
+      - python: "pypy3.5"
+        env: ANYBLOK_DATABASE_DRIVER=postgresql+psycopg2cffi
+        addons:
+            postgresql: "10"
+      - addons:
+            postgresql: "9.6"
+      - addons:
+            postgresql: "10"
```

Comment:
why postgresql 10 is there twice, should we test against v11 ?

Reply:
the avalaible value are 

postgresql | 9.4 9.5 9.6 10